src/model_finetuning/metric.py

```diff
@@ -0,0 +1,22 @@
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+
+class MCRMSELoss(nn.Module):
```

Comment:
I'd better rename this file to losses.py, because in the future we will try different loss function - e.g. I have seen kernel that tells SmoothL1 loss works better than original loss, also I have seen kernels that use simple MSE loss without mean column root part.